### ISSUE TITLE
[Feature] Entry Fields To Add For A Reviewer's Experience/Background for Context

### ISSUE THREAD

**Issue by advra** [martzcodes/OMSCentral#84]
After using omscentral for the incoming 3rd semester I found the reviews to be widely inaccurate due to the lack of an individuals background. A personal may have no professional experience but enough side projects to warrant an A. Specifically one can say "it was an easy class despite not having any experience" or "I have a lot of experience in the software industry but found this relatively difficult". I believe adding these fields can really improve the quality of these reviews. A structure to normalize the content can help me and others read through reviews quicker rather than reading long paragraphs and extracting this type of information to determine if the review would be relevant to the reader. 

Of course they can leave this info blank if they wish but the goal is to organize not enforce.

I look forward to your response.

 Proposed Structured Comment Example: 
> **Comment:** [Some generic paragraph review about the class here. Some generic paragraph review about the class here.Some generic paragraph review about the class here.Some generic paragraph review about the class here.Some generic paragraph review about the class here.Some generic paragraph review about the class here.Some generic paragraph review about the class here.Some generic paragraph review about the class here.Some generic paragraph review about the class here.Some generic paragraph review about the class here.Some generic paragraph review about the class here.Some generic paragraph review about the class here.Some generic paragraph review about the class here.Some generic paragraph review about the class here.Some generic paragraph review about the class here.Some generic paragraph review about the class here.Some generic paragraph review about the class here.Some generic paragraph review about the class here.]
> **Years of Software Experience:** 3
> **Industries:** Semiconductor (1), Defense (3)
> **Languages:** C++ (1), Java (3)

**Comment by ADustyOldMuffin** [comment]
I think a really good field would be what your undergrad was. So IE it'd be helpful if you could see if the person saying that a class that requires I lot of CS concepts is difficult, but their undergrad was a non CS degree or a math intensive class is being reviewed as easy by someone with a math related undergrad.

I think that mixed with a simple background would be pretty good. Might also be a good idea to save a user's background so they don't have to enter it every time (the less you make people do the more likely they are to use it).

**Comment by janmarander** [comment]
I agree with this assessment.  I noticed many people inflating the average number of hours spent on a class because they were also learning the programming language the course was using.  It would be helpful to know this because if you are already well versed in that language, obviously your time spent on the course will be lower.
